### PR TITLE
PLATFORM-1331 MercuryImageReview cleanup iframe styles

### PR DESCRIPTION
**Pull request by pawelchcki**
- add temporary hack for iframe height 
- Fix translations and description.

/cc: @jcellary @wladekb @RafalWilinski

### CODE REVIEW

File: extensions/wikia/ImageReviewMercury/templates/ImageReviewMercury_index.php [hunk 1]
```diff
@@ -1,2 +1 @@
-<iframe src="/image-review?useskin=wikiamobile&fullscreen=true" id="ImageReviewToolIframe"' +
-'class="curated-content-tool" name="curated-content-tool" style="height:100vw; width: 100vh;"></iframe>
\ No newline at end of file
+<iframe src="/image-review?useskin=wikiamobile&fullscreen=true" id="ImageReviewToolIframe" width="100%" style="height:300vw"></iframe>
```
Comment (RafalWilinski) [inline extensions/wikia/ImageReviewMercury/templates/ImageReviewMercury_index.php:1]: `&` => `&amp;`

File: extensions/wikia/ImageReviewMercury/ImageReviewMercuryController.class.php [hunk 2]
```diff
@@ -4,7 +4,6 @@ class ImageReviewMercuryController extends WikiaSpecialPageController {
 
 	private $imageReview;
 	public function __construct() {
-
 		$this->imageReview = (new ImageReviewMercury);
```
Comment (RafalWilinski) [inline extensions/wikia/ImageReviewMercury/ImageReviewMercuryController.class.php]: Parenthesees are not needed here

**Comment by RafalWilinski** [comment]
Despite 2 minor issues LGTM

**Comment by RafalWilinski** [comment]
Closed accidentally by missclick, sorry.

File: extensions/wikia/ImageReviewMercury/templates/ImageReviewMercury_index.php (hunk 1 above)
Comment (pawelchcki) [inline extensions/wikia/ImageReviewMercury/templates/ImageReviewMercury_index.php:1]: i think & needs to stay, it looks like its incorrectly flagged by github

**Pull request merged**